src/wallet/wallet2.cpp

```diff
@@ -3508,6 +3686,16 @@ std::string wallet2::make_multisig(const epee::wipeable_string &password,
 
   clear();
 
+  // decrypt keys
+  if (m_ask_password)
```

Comment:
I believe this needs to be `if (m_ask_password && !m_rpc)`. Also, the current patch would make the rest of the process completely skipped when the encryption mode is disabled. It should instead be:
```c++
  // decrypt keys
  epee::misc_utils::auto_scope_leave_caller keys_reencryptor;
  if (m_ask_password && !m_rpc)
  {
    ...
    keys_reencryptor = epee::misc_utils::create_scope_leave_handler(...);
  }
```
The same for `finalize_multisig`.